lib/sqlite.js

```diff
@@ -1,5 +1,40 @@
 'use strict';
 const { emitExperimentalWarning } = require('internal/util');
+const {
```

Comment:
Even though this is still marked as a draft, I wanted to leave a comment before you go too far down this path.

Right now the `node:sqlite` module has no JavaScript code. I think we should aim to keep it that way.

Reply:
I appreciate that, @cjihrig . I will keep it on the C++ side only then.